### DISCUSSION TITLE
Paraview: 1st order variable on 2nd order mesh

### DISCUSSION THREAD

**Discussion by joe61vette** [idaholab/moose#21848] [Q&A General] [answered]
Hello:

Trying to visualize a 1st order variable on a 2nd order mesh (some vars are 2nd order).  I get unphysical values for the 1st order variables at the 2nd order node locations.  For example:

![T_fluid](https://user-images.githubusercontent.com/8108870/184656465-4fe4b24e-a403-4275-9907-0fc48cfb5f20.png)

Is there a setting in Paraview to force interpolation of the variables? (I have asked on the Paraview forum but no response yet). Or do I need to create 2nd order auxvars for my 1st order vars?

Thanks,
  Joe Kelly

**Comment by GiudGiud** [comment]
Hi Joe

I've never seen that.
@roystgnr @dschwen have you / do you have a solution?

Guillaume

**Comment by roystgnr** [comment, marked as answer]
I've never seen this either.  I can at least say that "Or do I need to create 2nd order auxvars for my 1st order vars?" isn't the long-term solution - there has to be some kind of bug here.

Is there any way you can reduce the problem to an input file we can use to replicate it?

**Reply by joe61vette** [reply]
Thanks.  I'll do that later today.
